pkg/lib/authn/identity/info.go

```diff
@@ -75,9 +75,10 @@ func (i *Info) ToSpec() Spec {
 		return Spec{
 			Type: i.Type,
 			SIWE: &SIWESpec{
-				VerificationRequest: model.SIWEVerificationRequest{
-					Message:   i.SIWE.Data.Message,
-					Signature: i.SIWE.Data.Signature,
+				VerifiedData: model.SIWEVerifiedData{
```

Comment:
I guess we need to revert this commit as we are going to combine verification and form submission into a single step.